src/applications/proxy-rewrite/redirects/crossDomainRedirects.json

```diff
@@ -478,5 +478,55 @@
     "domain": "www.benefits.va.gov",
     "src": "/homeloans/adaptedhousing.asp",
     "dest": "/housing-assistance/disability-housing-grants/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/homeloans/resources_veteran.asp",
+    "dest": "/housing-assistance/home-loans/home-buying-process/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/homeloans/purchaseco_loan_fee.asp",
+    "dest": "/housing-assistance/home-loans/funding-fee-and-closing-costs/"
+  },
+  {
+    "domain": "www.benefits.va.gov",
+    "src": "/homeloans/purchaseco_loan_limits.asp",
+    "dest": "/housing-assistance/home-loans/loan-limits/"
+  },
```

Comment:
End of https://github.com/department-of-veterans-affairs/va.gov-team/issues/2214